modules/id5IdSystem.js

```diff
@@ -58,11 +58,13 @@ export const id5IdSubmodule = {
    * @returns {IdResponse|undefined}
    */
   getId(configParams, consentData, cacheIdObj) {
-    if (!hasRequiredParams(configParams)) {
+    // This check is redundant. Is being handled at line #65. - UOE-5788
+    /* if (!hasRequiredParams(configParams)) {
       return undefined;
-    }
+    } */
     if (!configParams || typeof parseInt(configParams.partner) !== 'number') {
```

Comment:
Let's remove this extra check and put the parseInt condition in hasRequiredParams

Reply:
Moved the changes in hasRequiredParams function and added a check for isNaN